css/app.css

```diff
@@ -82,6 +82,32 @@ body {
   margin: 20px 0;
 }
 
+#tone-up-btn {
+  background: #196127;
```

Comment:
It's better to use a class and write the shared style of these two buttons in a class.
And about the background color, it's better to use gradient because I use gradient in other buttons.